Plugins/DotNET/DotNETExports.cpp

```diff
@@ -538,27 +603,31 @@ std::vector<void*> GetExports()
     exports.push_back((void*)&ClosureAssignCommand);
     exports.push_back((void*)&ClosureDelayCommand);
     exports.push_back((void*)&ClosureActionDoCommand);
-    exports.push_back((void*)&nwnxSetFunction);
-    exports.push_back((void*)&nwnxPushInt);
-    exports.push_back((void*)&nwnxPushFloat);
-    exports.push_back((void*)&nwnxPushObject);
-    exports.push_back((void*)&nwnxPushString);
-    exports.push_back((void*)&nwnxPushString); // reserved utf8
-    exports.push_back((void*)&nwnxPushEffect);
-    exports.push_back((void*)&nwnxPushItemProperty);
-    exports.push_back((void*)&nwnxPopInt);
-    exports.push_back((void*)&nwnxPopFloat);
-    exports.push_back((void*)&nwnxPopObject);
-    exports.push_back((void*)&nwnxPopString);
-    exports.push_back((void*)&nwnxPopString); // reserved utf8
-    exports.push_back((void*)&nwnxPopEffect);
-    exports.push_back((void*)&nwnxPopItemProperty);
-    exports.push_back((void*)&nwnxCallFunction);
+    exports.push_back((void*)&NWNXSetFunction);
+    exports.push_back((void*)&NWNXPushInt);
+    exports.push_back((void*)&NWNXPushFloat);
+    exports.push_back((void*)&NWNXPushObject);
+    exports.push_back((void*)&NWNXPushString);
+    exports.push_back((void*)&NWNXPushString); // reserved utf8
```

Comment:
Rather than adding new ones, you should re-use these four reserved slots. This is exactly what they were for.